packages/sdk/src/fetcher/core/index.ts

```diff
@@ -188,51 +188,10 @@ class Fetcher implements ICoreFetcher {
         return { ...cachedTokens, ...fetchedTokens };
     }
 
-    private async fetchContentFromIPFSWithLocalStorageCache(
-        ipfsGatewayURL: string,
-        cacheableCids: string[]
-    ) {
-        const cachedCids: { [cid: string]: string } = {};
-        const uncachedCids = [];
-        for (const cid of cacheableCids) {
-            const cachedContent = CACHER.get<string>(cid);
-            if (!!cachedContent) cachedCids[cid] = cachedContent;
-            else uncachedCids.push(cid);
-        }
-        if (uncachedCids.length > 0) {
-            const uncachedContent = await Promise.all(
-                uncachedCids.map(async (cid) => {
-                    const response = await fetch(
-                        `${ipfsGatewayURL}/ipfs/${cid}`
-                    );
-                    const responseOk = response.ok;
-                    warn(responseOk, `could not fetch content with cid ${cid}`);
-                    return {
-                        cid,
-                        content: responseOk ? await response.text() : null,
-                    };
-                })
-            );
-            for (const { cid, content } of uncachedContent) {
-                if (!content) continue;
-                cachedCids[cid] = content;
-                CACHER.set(cid, content, Number.MAX_SAFE_INTEGER);
```

Comment:
Build is failing, the `CACHER` import needs to be removed:
```
src/fetcher/core/index.ts:6:5 - error TS6133: 'CACHER' is declared but its value is never read
```

Reply:
Addressed in 13cefd5dbb020d21f97414584fcfc215b64a3a72